src/postman/transformers/response.ts

```diff
@@ -0,0 +1,42 @@
+import { HttpParamStyles, IHttpHeaderParam, IHttpOperationResponse } from '@stoplight/types/dist';
+import { Cookie, Response } from 'postman-collection';
+import { transformDescriptionDefinition } from '../util';
+import { transformHeader, transformRawBody } from './request'; // @todo move me to common!
+
+export function transformResponse(response: Response): IHttpOperationResponse {
+  const headers = response.headers.map(transformHeader);
+  const mediaType = response.headers.get('content-type');
+
+  return {
+    code: String(response.code),
+    description: response.description && transformDescriptionDefinition(response.description),
+    headers: headers.concat(response.cookies.map(transformCookie).filter((c: Cookie) => c)),
+    contents: mediaType && [transformRawBody(response.body, mediaType)],
```

Comment:
```suggestion
    contents: mediaType && response.body && [transformRawBody(response.body, mediaType)],
```

so we can remove the optional body in the `transformRawBody` function

Reply:
👍